ModTek/ModTek.cs

```diff
@@ -113,29 +114,40 @@ private static void PropagateConflictsForward(Dictionary<string, ModDef> modDefs
             }
         }
 
-        private static Queue<ModDef> GetLoadOrder(Dictionary<string, ModDef> modDefs, out List<string> unloaded)
+        internal static Queue<ModDef> GetLoadOrder(Dictionary<string, ModDef> modDefs, out List<string> skippedMods)
         {
             var loadOrder = new Queue<ModDef>();
-            var loaded = new HashSet<string>();
-            unloaded = modDefs.Keys.OrderByDescending(x => x).ToList();
-
-            int removedThisPass;
+            var loadedMods = new HashSet<string>();
+            var candidateMods = new SortedDictionary<string, ModDef>(modDefs);
+
+            //// remove all mods that can't ever be loaded due to missing dependency
+            // not necessary as load order issues have to be fixed by the user anyway
+            // only helps with allowing conflicting mods to load in case the offender can't be loaded due to dependency issues
+            //candidateMods.Values.ToList() // iterate over copy (ToList) to allow removal in original
+            //    .Where(m => !m.DependsOn.All(modDefs.ContainsKey))
+            //    .Do(m => candidateMods.Remove(m.Name));
+
+            // remove all mods who have conflicts, in order to be fair, remove all mods that conflict and don't pick a winner
```

Comment:
I believe one should even remove mods from being loaded if they are mentioned in a ConflictsWith by another mod. That is not part of the code yet.